flapper-news/public/javascripts/posts/posts.factory.js

```diff
@@ -1,14 +1,14 @@
 // needed to inject $http
-app.factory('posts', ['$http', [function($http){
-	
+app.factory('posts', ['$http', function($http){
+
 	var o = {
 		posts: [],
 	};
-	
+
 	// to retrieve posts in the posts service within angularApp.js
-	o.getall = function(){
+	o.getAll = function(){
 		return $http.get('/posts').success(function(data){
-			angular.copy(data. o.posts);
+			angular.copy(data, o.posts);
```

Comment:
Another one that looks right at first glance, I probably reloaded the page and looked at the error and looked at the page a half a dozen times before I actually saw the problem.

The error that was coming up was : `TypeError: Cannot read property 'posts' of undefined` and I was consoling out `o` and it was certainly defined.  Then I noticed the period after `data`, so it was trying to get `data.o.posts`, `data` is defined but it doesn't an `o` property ;)